src/pool/cli.c

```diff
@@ -290,8 +291,10 @@ process_query_reply(struct dc_pool *pool, struct pool_buf *map_buf,
 		struct pool_target     *ts;
 		int			i;
 
-		if (info != NULL)
+		if (info != NULL) {
+			bits = info->pi_bits;
 			memset(info, 0, sizeof(*info));
+		}
```

Comment:
It looks not friendly to the caller of daos_pool_query(), imagine the caller specified certain bits in the daos_pool_info_t parameter of daos_pool_query(), after daos_pool_query() returned, how the caller tell if the space info or rebuild status is returned? (the bits in daos_pool_info_t is cleared here)
My suggestion is:
- Either specify the 'bits' in a separate parameter (rather than in daos_pool_info_t) of daos_pool_query(), or;
- Don't touch the specified 'bits' in daos_pool_info_t here, or;
- Let server returns valid 'bits' in reply, propagate it to caller via the 'bits' in daos_pool_info_t;

Reply:
agreed, I'll update it.